letsencrypt-apache/letsencrypt_apache/configurator.py

```diff
@@ -88,20 +82,32 @@ class ApacheConfigurator(augeas_configurator.AugeasConfigurator):
 
     description = "Apache Web Server - Alpha"
 
+    # Kept in same function to avoid multiple compilations of the regex
+
+    private_ips_regex = re.compile(
+        r"(^127\.0\.0\.1)|(^10\.)|(^172\.1[6-9]\.)|"
+        r"(^172\.2[0-9]\.)|(^172\.3[0-1]\.)|(^192\.168\.)")
+    hostname_regex = re.compile(
+        r"^(([a-z0-9]|[a-z0-9][a-z0-9\-]*[a-z0-9])\.)*[a-z]+$", re.IGNORECASE)
+
+
     @classmethod
     def add_parser_arguments(cls, add):
-        add("server-root", default=constants.CLI_DEFAULTS["server_root"],
-            help="Apache server root directory.")
         add("ctl", default=constants.CLI_DEFAULTS["ctl"],
-            help="Path to the 'apache2ctl' binary, used for 'configtest' and "
-                 "retrieving Apache2 version number.")
+            help="Path to the 'apache2ctl' binary, used for 'configtest', "
+                 "retrieving the Apache2 version number, and initialization "
+                 "parameters.")
         add("enmod", default=constants.CLI_DEFAULTS["enmod"],
             help="Path to the Apache 'a2enmod' binary.")
+        add("dismod", default=constants.CLI_DEFAULTS["dismod"],
```

Comment:
Oh sorry, I thought that `constants.CLI_DEFAULTS['enmod'] == constants.CLI_DEFAULTS['dismod'] == 'a2enmod'`. My bad!